src/libpriv/rpmostree-bwrap.c

```diff
@@ -231,6 +231,33 @@ rpmostree_bwrap_new (int rootfs_fd,
   if (!running_in_nspawn ())
     rpmostree_bwrap_append_bwrap_argv (ret, "--unshare-net", NULL);
 
+  /* Capabilities; this is a subset of the Docker (1.13 at least) default.
+   * Specifically we strip out in addition to that:
+   *
+   * "cap_net_raw" (no use for this in %post, and major source of security vulnerabilities)
+   * "cap_mknod" (%post should not be making devices, it wouldn't be persistent anyways)
+   * "cap_audit_write" (we shouldn't be auditing anything from here)
+   *
+   * But crucially we're dropping a lot of other capabilities like
+   * "cap_sys_admin", "cap_sys_module", etc that Docker also drops by default.
+   * We don't want RPM scripts to be doing any of that. Instead, do it from
+   * systemd unit files.
+   */
+  if (getuid () == 0)
+    rpmostree_bwrap_append_bwrap_argv (ret, "--cap-drop", "ALL",
+                                       "--cap-add", "cap_chown",
+                                       "--cap-add", "cap_dac_override",
+                                       "--cap-add", "cap_fowner",
+                                       "--cap-add", "cap_fsetid",
+                                       "--cap-add", "cap_kill",
+                                       "--cap-add", "cap_setgid",
+                                       "--cap-add", "cap_setuid",
+                                       "--cap-add", "cap_setpcap",
+                                       "--cap-add", "cap_net_bind_service",
```

Comment:
Can we drop this one too?

Reply:
Yeah, we should.  I originally was worried about apps which use TCP for local management...IIRC `bind` at least used to do this?  But OTOH we already break this by using an isolated loop-only netns so...yeah, let's remove it.